README.md

```diff
@@ -10,31 +10,29 @@ This implementation is based directly on the algorithm published in the Felzensz
 Demo: http://mapbox.github.io/tiny-sdf/
 
 ## Usage
+
```

Comment:
Could the README have a little setup section? Maybe:

- Showing travis badge
- Listing node version dependency (I happened to be on node v10 the first time I tried `npm test` and got an unfriendly "unexpected identified" error from the "import TinySDF" line)
- Pointing to `npm install` and `npm test`